packages/happn-db-provider-mongo/test/__fixtures/data/functional_mixed.nedb

```diff
@@ -1,13 +1,13 @@
 {"snapshot":"{\"filename\":\"loki.db\",\"collections\":[{\"name\":\"happn\",\"data\":[],\"idIndex\":null,\"binaryIndices\":{\"path\":{\"name\":\"path\",\"dirty\":false,\"values\":[]},\"created\":{\"name\":\"created\",\"dirty\":false,\"values\":[]},\"modified\":{\"name\":\"modified\",\"dirty\":false,\"values\":[]}},\"constraints\":null,\"uniqueNames\":[\"path\"],\"transforms\":{},\"objType\":\"happn\",\"dirty\":true,\"cachedIndex\":null,\"cachedBinaryIndex\":null,\"cachedData\":null,\"adaptiveBinaryIndices\":true,\"transactional\":false,\"cloneObjects\":false,\"cloneMethod\":\"parse-stringify\",\"asyncListeners\":false,\"disableMeta\":false,\"disableChangesApi\":true,\"disableDeltaChangesApi\":true,\"autoupdate\":false,\"serializableIndices\":true,\"disableFreeze\":true,\"ttl\":null,\"maxId\":0,\"DynamicViews\":[],\"events\":{\"insert\":[],\"update\":[],\"pre-insert\":[],\"pre-update\":[],\"close\":[],\"flushbuffer\":[],\"error\":[],\"delete\":[null],\"warning\":[null]},\"changes\":[],\"dirtyIds\":[]}],\"databaseVersion\":1.5,\"engineVersion\":1.5,\"autosave\":false,\"autosaveInterval\":5000,\"autosaveHandle\":null,\"throttledSaves\":true,\"options\":{\"serializationMethod\":\"normal\",\"destructureDelimiter\":\"$<\\n\"},\"persistenceMethod\":\"fs\",\"persistenceAdapter\":null,\"verbose\":false,\"events\":{\"init\":[null],\"loaded\":[],\"flushChanges\":[],\"close\":[],\"changes\":[],\"warning\":[]},\"ENV\":\"NODEJS\"}"}

```

Comment:
check why nedb extension is used here - this is a loki db, find out whether we want this in code base

Reply:
Looking at the test I believe it should be a Loki DB. Renamed accordingly, and made db-provider explicit in test